hub/integrations/pytorch/pytorch.py

```diff
@@ -178,6 +182,9 @@ def __getitem__(self, index: int):
             self._all_shared_memory_clean_up()
             self.processed_range = slice(-1, -1)
 
+        if sample is None:
+            return None
```

Comment:
returning None could be problematic in many cases. Can we return 0th index's data (or any index that had valid data) instead?